common/config/environment.go

```diff
@@ -26,6 +26,7 @@ const (
 	DDInfraOSFamily          = "osFamily"
 	DDInfraOSArchitecture    = "osArchitecture"
 	DDInfraOSAmiID           = "osAmiId"
+	DDInfraResourcesTags     = "resourcesTags"
```

Comment:
Something like `extraResourceTags` is probably more accurate as this option does not replace default resource tags.

Reply:
It overwrite tags with the same key (ie if you pass `managed-by:totoro` it will replace `managed-by:pulumi`